litgpt/finetune/adapter_v2.py

```diff
@@ -178,6 +178,12 @@ def main(
     if fabric.device.type == "cuda":
         fabric.print(f"Memory used: {torch.cuda.max_memory_allocated() / 1e9:.02f} GB")
 
+    # Final evaluation
+    val_loss = validate(fabric, model, val_dataloader, dataclasses.replace(eval, max_iters=len(val_dataloader)))
+    metrics = {"val_loss": val_loss, "val_ppl": math.exp(val_loss)}
+    fabric.log_dict(metrics)
+    fabric.print(f"Final evaluation | val loss: {val_loss.item():.4f} | val ppl: {math.exp(val_loss):.4f}")
```

Comment:
```suggestion
    fabric.print(f"Final evaluation | val loss: {val_loss.item():.4f} | val perplexity: {math.exp(val_loss):.4f}")
```

Reply:
It's not what we log though. I'd like to keep the PR focused on doing one thing only. So if it needs to be renamed, let's do it separately.

Reply:
ok that's fine